bin.js

```diff
@@ -16,7 +16,9 @@ var opts = {
   force:     args.f || args.force,
   sockets:   args.s || args.sockets,
   verbose:   args.verbose === undefined ? process.stdin.isTTY : args.verbose,
-  frequency: args.frequency ? +args.frequency : null
+  frequency: args.frequency ? +args.frequency : null,
+  proxy:     args.proxy ? args.proxy : null,
+  strictSSL: args['strict-ssl'] !== null ? args['strict-ssl'] !== 'false' : null
```

Comment:
So it would look like `strictSSL: args['no-strict-ssl'] !== null` ?
(I forgot I made the dev some months ago and I added strictSSL option because of some entreprise network problems)